src/runtime_src/core/tools/common/ReportAie.cpp

```diff
@@ -132,6 +132,13 @@ ReportAie::writeReport(const xrt_core::device* /*_pDevice*/,
 
 	if(tile.second.find("dma") != tile.second.not_found()) {
           _output << boost::format("    %s:\n") % "DMA";
+
+          _output << boost::format("        %s:\n") % "FIFO";
+          for(auto& node : tile.second.get_child("dma.fifo.counters")) {

```

Comment:
Being that the "counter_id" has been requested to be removed from the data structure, I do understand that this for loop needs to be re-examined.